crates/chain/tests/test_tx_graph.rs

```diff
@@ -149,14 +153,17 @@ fn insert_tx_can_retrieve_full_tx_from_graph() {
         output: vec![TxOut::default()],
     };
 
-    let mut graph = TxGraph::default();
+    let mut graph = TxGraph::<BlockId>::default();
```

Comment:
`()` should be used as the anchor in these tests unless they are actually testing anchor functions.

Reply:
Done in 67a4abae85b1bdaf16092a753f5f2a5f8726d769